.github/workflows/release.yml

```diff
@@ -38,6 +38,9 @@ jobs:
 
       # No special setup for Go.
 
+      # Rust: publish with the oldest supported release to ensure compatible lockfiles etc.
+      - run: rustup default 1.63 && rustup component add clippy rustfmt
```

Comment:
🤔 why would lockfile matter for publishing? We _shouldn't_ be publishing it, no? But, yeah, using the oldest makes sense to me for general reasons (if it works on oldest, it will work on newest for sure)